code/datums/materials/basemats.dm

```diff
@@ -270,6 +270,18 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	qdel(source.GetComponent(/datum/component/hot_ice, "plasma", amount*150, amount*20+300))
 	return ..()
 
+/datum/material/metalhydrogen
+	name = "Metal Hydrogen"
+	id = "metal hydrogen"
+	desc = "Solid metallic hydrogen. Some say it should be impossible"
+	color = "#f2d5d7"
+	alpha = 150
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/metal_hydrogen
+	value_per_unit = 0.75
+	beauty_modifier = 0.75
```

Comment:
too strong too

Reply:
same

Reply:
being hard to make != being pretty.

Reply:
but is a pretty material

Reply:
stop resolving comments if you don't actually adress the comment and simply disagree. either change it or leave it open so a maintainer can make a judgement call

Reply:
you too don't leave helpful comments, "too strong too" what does it mean? too strong for what, too strong compared to what, why an end game item should not be strong, what is an example of good strong? if you leave half assed comment without telling what is wrong and why is wrong, dont complain later

Reply:
this is 50% better than an admin-only material.

if this was more in line with other materials 0.3 for example, it'd be better.

Reply:
nerfed